crates/bevy_ui/Cargo.toml

```diff
@@ -21,7 +21,9 @@ bevy_image = { path = "../bevy_image", version = "0.17.0-dev" }
 bevy_input = { path = "../bevy_input", version = "0.17.0-dev" }
 bevy_math = { path = "../bevy_math", version = "0.17.0-dev" }
 bevy_reflect = { path = "../bevy_reflect", version = "0.17.0-dev" }
-bevy_sprite = { path = "../bevy_sprite", version = "0.17.0-dev" }
+bevy_sprite = { path = "../bevy_sprite", version = "0.17.0-dev", features = [
+  "bevy_text",
```

Comment:
to be clear here, ui doesnt need text2d, right, it just enables this because it unconditionally enables both text and sprite?

Reply:
Yeah that's right, it'd be good to disentangle them somehow later.